### PR TITLE
Consolidate browsers/permissions/accounts/wallet db in a single database

### LINKED ISSUE #1536: Consolidate browsers/settings/dapps permission in a single not-network specific database
# Problem

Opening each encrypted database adds overhead to a node startup procedure. At the same time all specified databases can be merged into a single database with little effort.

# Implementation

Merge migration to a single directory. Initialize database once and then pass it to all services that use this database.

# Acceptance Criteria

The startup became faster. Maybe add some kind of benchmark for node startup.

### PR DESCRIPTION
**Pull request by dshulyak**
closes: https://github.com/status-im/status-go/issues/1536

### CODE REVIEW

**Review by adambabik — APPROVED** [review]

File: services/wallet/database.go [hunk 1]
```diff
@@ -111,6 +96,10 @@ func (blob *JSONBlob) Value() (driver.Value, error) {
 	return json.Marshal(blob.data)
 }
 
+func NewDB(db *sql.DB) *Database {
+	return &Database{db}
```
Comment (adambabik) [inline services/wallet/database.go]: With this PR we are vulnerable to the problem described in https://github.com/status-im/status-go/issues/1537?

Comment (dshulyak) [reply services/wallet/database.go]: yes that part wasn't added yet

Comment (dshulyak) [reply services/wallet/database.go]: @adambabik made a separate pr to resolve that problem (https://github.com/status-im/status-go/pull/1564)

**Comment by yenda** [comment]
what is the status on this one? can it be merged?

**Comment by dshulyak** [comment]
@yenda it is based on accounts-persistence branch. once we will merge that one - i will start merging all smaller changes

**Pull request merged**